addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -277,6 +277,20 @@ def _l10n_es_edi_get_invoices_info(self, invoices):
 
             invoice_node['DescripcionOperacion'] = invoice.invoice_origin[:500] if invoice.invoice_origin else 'manual'
             if invoice.is_sale_document():
+                # OSS taxes
+                is_oss = False
+                company = invoice.company_id
+                country = company.account_fiscal_country_id
+                oss_tax_groups = self.env['ir.model.data'].search([
+                    ('module', '=', 'l10n_eu_oss'),
+                    ('model', '=', 'account.tax.group')])
+                oss_taxes = self.env['account.tax'].search([
+                    ('company_id', '=', company.id),
+                    ('country_id', '=', country.id),
+                    ('tax_group_id', 'in', oss_tax_groups.mapped('res_id'))])
+                for line in invoice.invoice_line_ids.filtered(lambda line: line.display_type not in ('line_section', 'line_note')):
+                    if not is_oss and any([t in oss_taxes for t in line.tax_ids]):  # noqa: C419
```

Comment:
and t.group_id in oss_tax_groups.mapped('res_id') something

also if one is found we could break
and we could do invoice.invoice_line_ids.mapped('tax_ids.group_id') immediately and then intersect (something nice)

Reply:
Instead of checking the oss taxes, I've made an intersect between `invoice.invoice_line_ids.mapped('tax_ids.tax_group_id').ids` and `oss_tax_groups.mapped('res_id')`